### DISCUSSION TITLE
Remove Mob from Transformation List

### DISCUSSION THREAD

**Discussion by dra3d0n** [Geolykt/EnchantmentsPlus#130] [Q&A]
Hello! I was wondering how do you blacklist/remove a mob from the Transformation enchant. I'm planning to remove the ability to turn endermen into shulkers, if it's possible. 

Great plugin, me and my friends are having a blast using it!

Thanks

**Comment by Geolykt** [comment]
The magic configuration file allows to do that, see https://github.com/Geolykt/EnchantmentsPlus/blob/4xx/resources/magicCompat.yml#L464

**Reply by dra3d0n** [reply]
Thank you so much!
